web/styles/compose.css

```diff
@@ -844,7 +844,6 @@ textarea.new_message_textarea {
 
     &:read-only,
     &:disabled {
-        cursor: not-allowed;
```

Comment:
I did not find any way in js, we reach either the disabled or read-only state for new_message_textarea. I'm not too confident on removing this, so I've let it be for now. But if we know that we will not end up in this state, might be just better to remove it.

Reply:
I think this might be possible if you're sending a message that cannot be locally echoed, like one containing a link to an externally hosted image -- we hold the compose box while waiting for the message to be confirmed by the server. Could be wrong.

Reply:
OK, the answer is that while we do hold the compose box, we don't disable the input.

So we can merge this, but would maybe add something like this back in the event that we change the above.